tests/unitTestsNotUnderCtest/testDoubleBufferingHW.cc

```diff
@@ -0,0 +1,125 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE SubdeviceBackendUnifiedTest
```

Comment:
Copy/paste error: This is not a unified backend test

Reply:
fixed